logger.go

```diff
@@ -27,65 +27,84 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"path/filepath"
+	"strings"
 )
 
 type LoggingClient struct {
-	owningServiceName string
-	RemoteUrl         string
-	LogFilePath       string
-	stdOutLogger      *log.Logger
-	fileLogger        *log.Logger
+	OwningServiceName string
+	RemoteEnabled     bool
+	LogTarget         string
+	StdOutLogger      *log.Logger
+	FileLogger        *log.Logger
 }
 
 // Create a new logging client for the owning service
-func NewClient(owningServiceName string, remoteUrl string) LoggingClient {
+func NewClient(owningServiceName string, isRemote bool, logTarget string) LoggingClient {
```

Comment:
I think it is more readable to keep two different fields to have the remoteUrl and the local log filename. And just enable local and/or remote just if they are not empty.

Reply:
However, in the configs both fields are both populated. Example from core-data-go:

"Loggingfile" : "./core-data.log",
"Loggingremoteurl" : "http://localhost:48061/api/v1/logs"

So an additional boolean is necessary.